src/browser_launchers/puppeteer_launcher.js

```diff
@@ -164,57 +155,16 @@ export function apifyOptionsToLaunchOptions(launchContext) {
  * For an example of usage, see the [Synchronous run Example](../examples/synchronous-run)
  * or the [Puppeteer proxy Example](../examples/puppeteer-with-proxy)
  *
- * @param {PuppeteerLaunchContext} [options]
- *   Optional settings passed to `puppeteer.launch()`. In addition to
- *   [Puppeteer's options](https://pptr.dev/#?product=Puppeteer&show=api-puppeteerlaunchoptions)
- *   the object may contain our own  {@link PuppeteerLaunchContext} that enable additional features.
+ * @param {PuppeteerLaunchContext} [launchContext]
+* All `PuppeteerLauncher` parameters are passed via an launchContext object.
```

Comment:
Would be nice to keep (and update) the info about how to pass Puppeteer launch options.